analyzers/LdapQuery/LdapQuery.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+#Author: @cyber_pescadito
+import ldap
+import json
+from cortexutils.analyzer import Analyzer
+
+class LdapQuery(Analyzer):
+    def __init__(self):
+        Analyzer.__init__(self)
+        self.ldap_address = self.get_param('config.LDAP Server Address', None, 'ldap_address is missing')
+        self.username = self.get_param('config.LDAP Username', None, 'username is missing')
+        self.password = self.get_param('config.LDAP Password', None, 'password is missing')
+        self.base_dn = self.get_param('config.base DN', None, 'base_dn is missing')
+        self.attributes = self.get_param('config.Attributes', None, 'Missing attributes list to report')
+        self.payload = self.get_param('data', None, 'username to search in LDAP is missing')
+
+    def summary(self,raw):
+        taxonomies = []
+        level = "info"
+        namespace = "LDAP"
+        predicate = "Query"
+        value = "Success"
+        taxonomies.append(self.build_taxonomy(level, namespace, predicate, value))
```

Comment:
Is there any useful information you can put as the value of that taxonomie? probably the `cn` or the `uid`?

Reply:
I agree it would be strange to not include "cn" value in attributes list to fetch from the LDAP, but if "cn" isn't declared in "attributes" list by the admin in cortex analyzer configuration it could break the summary. So i chosen to not include any value of LDAP there.
What is your point of view? should I change de program design to enforce "cn" attribute fetching ?

Reply:
No, the idea was to make that taxonomie meaningful/useful. 

If it will be always: `Ldap:Query=Success` then it's not worth it.

I would make a taxonomy like: `LdapQuery:cn=XXX` or `LdapQuery:ATTRIBUTE_NAME=ARRTIBUTE_VALUE` or nothing

Reply:
is value a mandatory attribute of taxonomies?

Reply:
Note sure, but the taxonomy aims to provide important info, without having to read the full report. You can probably show a taxonomy for important attributes if they exist?